docs/en/operation/User-CLI.md

```diff
@@ -1169,6 +1171,31 @@ This command will attach the database `hive_db_name` (of type `hive`) from the U
 When paths are mounted for `s3a://bucket1`, the mount option `aws.accessKeyId=abc` will be used,
 and when paths are mounted for `s3a://bucket2`, the mount option `aws.accessKeyId=123` will be used.
 
+### Glue UDB
+For `glue` udb type, there are some additional properties with the `-o` options:
+  * `udb-glue.aws.<MOUNT_PROPERTY>`: specify the mount option for the Glue udb, the mount options
+  are as follows:
+    * `region`: the glue aws region
+    * `catalog.id`: the aws catalog id
+    * `accesskey`: the aws access key id
+    * `secretkey`: the aws secret key
```

Comment:
I think this is how the properties should be called

```suggestion
  * `udb-glue.<UDB_PROPERTY>`: specify the UDB options for the Glue UDB. The options
  are as follows:
    * `aws.region`: the glue aws region
    * `aws.catalog.id`: the aws catalog id
    * `aws.accesskey`: the aws access key id
    * `aws.secretkey`: the aws secret key
```

Reply:
It makes sense, Fixed.